projects/tinkerbell/cluster-api-provider-tinkerbell/GIT_TAG

```diff
@@ -1 +1 @@
-v0.1.0
+77df39b0a63a28deb91030e6b2cd88b5cb75286a
```

Comment:
Tags would be better. Are yall working with them to cut an actually release/tag at some point?

Reply:
Regarding patches, 
- Most of the patches were PBNJ related which we don't need now. Aravind will work on adding new Rufio patches to CAPT after he implements it
- Few of them are upstreamed or need modification which we will be doing upstream only.

I am not aware of cutting a new release/tag, I will confirm with them.

Reply:
Yep we will cut a tag once all the changes are finalized. For now, we are just tracking the commits to test the kubified components